src/ecdsa/ot_based_ecdsa/test.rs

```diff
@@ -166,10 +167,10 @@ fn test_refresh() -> Result<(), Box<dyn Error>> {
     let participants = generate_participants(11);
     let max_malicious = 5;
     let threshold = max_malicious + 1;
-    let keys = run_keygen(&participants, threshold)?;
+    let keys = run_keygen(Scheme::OtBasedEcdsa, &participants, threshold)?;
```

Comment:
The "t" is capital OT refers to Oblivious Transfer.
```suggestion
    let keys = run_keygen(Scheme::OTBasedEcdsa, &participants, threshold)?;
```